src/utils.py

```diff
@@ -67,6 +75,8 @@ def close_instagram(device_id):
     print("Close Instagram app")
     os.popen("adb" + ("" if device_id is None else " -s " + device_id) +
              " shell am force-stop com.instagram.android").close()
+    print("Turning OFF screen")
+    os.popen("adb shell input keyevent 26")
```

Comment:
I would create a constant for this "magic number" 26 to improve readability. i.e. WHATTEVER_BUTTON_CODE="26"